src/transformers/models/informer/modeling_informer.py

```diff
@@ -1449,7 +1449,7 @@ def __init__(self, config: InformerConfig):
         self.layers = nn.ModuleList([InformerEncoderLayer(config) for _ in range(config.encoder_layers)])
         self.layernorm_embedding = nn.LayerNorm(config.d_model)
 
-        if config.distil is not None:
+        if config.distil:
```

Comment:
@kashif and this fix also.

before, even when `config.distil = False` the all condition was true.

Reply:
ah nice! yes